deployments/arbitrum/Arbitrum_SpokePool.json

```diff
@@ -1,5 +1,5 @@
 {
-  "address": "0x53dcC21887d54f41B049724b73a2195CE7343873",
+  "address": "0x7a4Ba159A34c1C49F712B5079a28121bbA0BFb7f",
```

Comment:
Not specifically related to this PR: one thing that would be cool: if we could have a way to verify that a deployed address matches the compiled code from the repo.

Might be tricky because the bytecode depends on constructor args, but would be cool to have an automation to do this because it's essentially impossible to verify manually.

Reply:
there's a way to do this with forge it hink